core-plugins/docs/GroupByAggregate-batchaggregator.md

```diff
@@ -12,6 +12,14 @@ Supports `Average`, `Count`, `First`, `Last`, `Max`, `Min`,`Sum`,`Collect List`,
 `Concat If`, `Logical And If`, `Logical Or If`, `Sum Of Squares If`, `Corrected Sum Of Squares If`
 as aggregate functions.
 
+Group By stages are now eligible to execute in BigQuery when BigQuery ELT Transformation Pushdown 
```

Comment:
Can you add a header for BigQuery ELT Transformation Pushdown? Just so it's a separate section of the doc.